app/retail/templates/contributor_landing.html

```diff
@@ -0,0 +1,101 @@
+{% comment %}
+  Copyright (C) 2018 Gitcoin Core
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published
+  by the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static cookielaw_tags %}
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    {% include 'shared/head.html' %}
+    {% include 'shared/cards.html' %}
+    <link rel="stylesheet" href="{% static "v2/css/landing_page.css" %}" />
+  </head>
+  <body class="index d-flex flex-column">
+    {% include 'shared/tag_manager_2.html' %}
+    {% cookielaw_banner %}
+    {% include 'shared/nav.html' %}
+    <div id="landing_page_wrapper">
+      <div class="container-fluid header header-contributor">
+        {% include 'landing/contributor/hero.html' %}
+      </div>
+
+      <div class="container-fluid p-5 benefits_container">
+        {% include 'landing/contributor/benefits.html' %}
+      </div>
+
+      <div class="container-fluid p-5 whatisit_container">
+        <div class="row mb-5">
+          <div class="col text-center">
+            <h1>{% trans "What is Gitcoin?" %}</h1>
+          </div>
+        </div>
+        <div class="row d-flex justify-content-center align-items-center whatisit mb-4 flex-wrap">
```

Comment:
@thelostone-mc I thought of it but we've different content in the landing pages and I am unsure how to pass custom HTML to a template. Can you help me with that?

Reply:
It would look along the lines of [this](https://github.com/gitcoinco/web/blob/master/app/retail/views.py#L36-L62) 

We'd the it as an array of image + text and render them in the contributor_landing and the funder_landing page

Both would have it's own version of `whatisit` object (need a better name 😂 )

If it's too much of a trouble -> I'll take care of it later just leave a `TODO` for me

Reply:
@thelostone-mc Thanks for the explanation. I've refactored the code to include a 'What is Gitcoin' template. Do let me know how it looks.